### ISSUE TITLE
sample test video can not be get

### ISSUE THREAD

**Issue by MichaelWU0726** [metalbubble/TRN-pytorch#93]
in file download.sh:
`wget -P sample_data -r -nH --cut-dirs=1 --no-parent --reject="index.html*"  http://relation.csail.mit.edu/data/bolei_juggling/`
it showed `403：Forbidden`

**Comment by shank-gyrus** [comment]
Same issue for me as well.
```
wget -r -nH --cut-dirs=1 --no-parent --reject="index.html*"  http://relation.csail.mit.edu/data/juggling_frames/
--2021-02-26 18:20:29--  http://relation.csail.mit.edu/data/juggling_frames/
Resolving relation.csail.mit.edu (relation.csail.mit.edu)... 128.30.195.26
Connecting to relation.csail.mit.edu (relation.csail.mit.edu)|128.30.195.26|:80... connected.
HTTP request sent, awaiting response... 403 Forbidden
2021-02-26 18:20:29 ERROR 403: Forbidden.

```

**Comment by MichaelWU0726** [comment]
> Same issue for me as well.
> 
> ```
> wget -r -nH --cut-dirs=1 --no-parent --reject="index.html*"  http://relation.csail.mit.edu/data/juggling_frames/
> --2021-02-26 18:20:29--  http://relation.csail.mit.edu/data/juggling_frames/
> Resolving relation.csail.mit.edu (relation.csail.mit.edu)... 128.30.195.26
> Connecting to relation.csail.mit.edu (relation.csail.mit.edu)|128.30.195.26|:80... connected.
> HTTP request sent, awaiting response... 403 Forbidden
> 2021-02-26 18:20:29 ERROR 403: Forbidden.
> ```

Have you solved it ? If done, can you tell me how did you solve it ?

**Comment by shank885** [comment]
@MichaelWU0726 No, the issue is with the resource access permission. I think public access to the data is disabled.

**Comment by MichaelWU0726** [comment]
> @MichaelWU0726 No, the issue is with the resource access permission. I think public access to the data is disabled.

so don't know if the owner of this repository provide the access

**Comment by DutchErwin** [comment]
Yup public access has been disabled.
I've made the test-files available in my fork if anyone is still looking for them: https://github.com/LFarmbot/TRN-pytorch

**Comment by tjnkyqcy** [comment]
> Yup public access has been disabled. I've made the test-files available in my fork if anyone is still looking for them: https://github.com/LFarmbot/TRN-pytorch

"something-something dataset or jester dataset or charades dataset", these dataset downloads show 503, do you have any other way these datasets can be downloaded that you can provide, thanks.
